media/css/custom.scss

```diff
@@ -0,0 +1,469 @@
+  color: $white;
+  text-decoration: none;
+  border-top: 2px solid $sidebar-border-color;
+  height: 20px;
+}
+
+#twitter ul.twits {
+  display: none;
+  margin: 0;
+  padding: 0;
+}
+
+#twitter_title {
+  padding: 6px;
+  color: $white;
+  font: 17px / 18px bold $font-stack;
+  text-decoration: none;
+  border-bottom: 2px solid $sidebar-border-color;
+}
+
+#twitter_bottom {
+  padding: 2px 6px;
+  background: $sidebar-background;
+  color: $white;
+  text-decoration: none;
+}
+
+.white {
+  color: $white;
+}
+
+.article {
+  .thumbnail {
+    margin: 15px;
+  }
+}
+
+a {
+  color: $sidebar-background;
+  &:hover {
+    color: $sidebar-link-highlight;
+  }
+  &:focus {
+    outline: thin dotted $sidebar-link-highlight;
+    outline-offset: -2px;
+  }
+}
+
+.well {
+  background-color: $sidebar-background;
+  li a {
+    color: $white;
+  }
+}
+
+a.thumbnail {
+  background-color: $white;
+  &:hover {
+    background-color: $sidebar-link-highlight;
+  }
+}
+
+#sidebar {
+  .well,
+  h2,
+  a {
+    color: $white;
+  }
+  a:hover {
+    color: $sidebar-link-highlight;
+  }
+}
+
+#panel1,
+#panel2,
+#panel3 {
+  display: none;
+  padding: 5px;
+  width: 225px;
+  height: 40px;
+  background: $sidebar-background;
+  color: $white;
+}
+
+#photo-feed {
+  > .span3 {
+    height: 350px;
+    margin: 5px 5px 25px;
+  }
+  .thumbnail {
+    height: inherit;
+    & > img {
+      width: 210px;
+      height: 200px;
+    }
+    .caption {
+      h4 {
+        height: 3em;
+        overflow: hidden;
+      }
+    }
+  }
+  .caption {
+    p {
+      font-size: 14px;
+    }
+  }
+}
+
+.twitter-follow-button {
+  margin-top: 3px;
+}
+
+.btn-danger {
+  background: $sidebar-link-highlight;
+  &:hover {
+    background: $danger-link-highlight;
+  }
+}
+
+h1,
+h2,
+h3 {
+  color: $dark-blue;
+}
+
+.leftimg,
+.deanimg {
+  height: 300px;
+}
+
+.leftimg {
+  float: left;
+  margin: 10px 10px 20px 0;
+}
+
+.deanimg {
+  margin: 20px 100px 20px 0;
+}
+
+.codehilite {
+  clear: left;
+}
+
+.badge {
+  padding-left: 6px;
+  padding-bottom: 3px;
+  font-size: 11px;
+}
+
+.badge-important {
+  background-color: $sidebar-link-highlight;
+}
+
+#loader {
+  background-color: $sidebar-link-highlight;
+  margin-bottom: 20px;
+  padding-bottom: 3px;
+}
+
+#spinner {
+  display: inline;
+  margin-right: 3px;
+}
+
+*::selection,
+*::-moz-selection,
+*::-webkit-selection {
+  background: $bright-red;
+  color: $white;
+}
+
+@import "social_buttons";
+@import "navigation";
+
+* {
+  // old-style reset here
+  border: 0;
+  padding: 0;
+}
+
+table {
+  border-collapse: separate;
+  border: 1px solid $grayish-blue;
+  border-width: 0 0 1px 1px;
+  font-size: 20px;
+}
+
+th {
+  text-align: center;
+  vertical-align: middle;
+  position: relative;
+}
+
+th {
+  font-weight: bold;
+  font-size: 14px;
+}
+
+td:hover,
+th:hover {
+  background-position: 0 -81px;
+  color: $light-black;
+}
+
+.events {
+  position: relative;
+  ul {
+    text-align: left;
```

Comment:
Properties should be ordered background, border, -webkit-box-shadow, box-shadow, color, color, display, font-size, list-style, padding, position, text-align, width, z-index